app/controllers/api/v1x2/order_processes_controller.rb

```diff
@@ -32,6 +32,16 @@ def destroy
 
         head :no_content
       end
+
+      def update_pre
+        order_process = OrderProcess.find(params.require(:order_process_id))
+        authorize(order_process, :update?)
+
+        pre_portfolio_item = PortfolioItem.find(params.require(:portfolio_item_id))
+        order_process.update!(:pre => pre_portfolio_item)
```

Comment:
@eclarizio I know it's not a lot of code here, but for consistency I think we wanted all the controller logic placed in service classes.  And I'm sensing we could use the same service class for `pre` and `post` with a simple parameter to specify which one was being set.